buildscripts/cmake/SetupBuildEnvironment.cmake

```diff
@@ -60,8 +60,8 @@ endif()
 
 # Mac-specific
 if(OS_IS_MAC)
-    set(MACOSX_DEPLOYMENT_TARGET 10.15)
-    set(CMAKE_OSX_DEPLOYMENT_TARGET 10.15)
+    # set(MACOSX_DEPLOYMENT_TARGET 10.15)
+    # set(CMAKE_OSX_DEPLOYMENT_TARGET 10.15)
```

Comment:
This really should not be necessary (note that deployment target means _minimum_ supported macOS version, not _only_ supported macOS version)